src/ProjectTemplates/Web.ItemTemplates/content/ApiController/ValueController.cs

```diff
@@ -0,0 +1,48 @@
+using Microsoft.AspNetCore.Http;
```

Comment:
Looks so much cleaner than the cshtml equivalents.

Reply:
yes it does